ts/theme/variables.ts

```diff
@@ -131,6 +134,16 @@ const customVariables = Object.assign(materialVariables, {
     return this.fontSize6 * this.lineHeightFontSizeRatio;
   },
 
+   //H2
```

Comment:
@fpersico is this ok?

Reply:
I implemented H2 to render the numbered list being it not used (I did not exist)